packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/template.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from '@microsoft/fast-element';
+import type { RichTextMentionUsersView } from '.';
+
+export const template = html<RichTextMentionUsersView>`<span
+    class="control"
+    part="control"
+    mention-href="${x => x.mentionHref}"
+    mention-label="${x => x.mentionLabel}"
```

Comment:
Why are you binding these attributes to the span element in the shadow root? The span element does not have those attributes, what is the goal here?

Reply:
I thought that the Tiptap requires these custom attributes to be part of internal control when rendering in the editor and getting and setting the markdown strings but it seems it is not required. This view element itself will act as a mention node for all purposes. So I have removed them.